### PR TITLE
Raiden

### PR DESCRIPTION
**Pull request by Kamanati**
# Pull Request Template

## Type of change

- [ ] Bug fix
- [x] Feature
- [ ] Documentation update

## Description

*ramble here*

## Checklist

- [x] any anime playing
- [x] bumped version
---
- [x] next, prev and replay work
- [x] `-c` history and continue work
- [x] `-d` downloads work
- [x] `-s` syncplay works
- [x] `-q` quality works
- [x] `-v` vlc works
- [x] `-e` select episode works
- [x] `-S` select index works
- [x] `-r` range selection works
- [x] `--skip` ani-skip works
- [x] `--skip-title` ani-skip title argument works
- [x] `--no-detach` no detach works
- [x] `--dub` and regular (sub) mode both work
- [x] `-L` language of anime title (jp, jl)
- [x] all providers return links (not necessarily on a single anime, use debug mode to confirm)
---
- [x] `-h` help info is up to date
- [x] Readme is up to date
- [x] Man page is up to date

## Additional Testcases

- The safe bet: One Piece
- Episode 0: Saenai Heroine no Sodatekata ♭
- Unicode: Saenai Heroine no Sodatekata ♭
- Non-whole episodes: Tensei shitara slime datta ken (ep. 24.5, ep. 24.9)

### CODE REVIEW

**Comment by Kamanati** [comment]
> # Pull Request Template
> ## Type of change
> * [ ]  Bug fix
> * [x]  Feature
> * [ ]  Documentation update
> * [ ] Added Additional interactive info

> ## Description
> _ramble here_
> 
> ## Checklist
> * [x]  any anime playing
> * [ ]  bumped version
> 
> * [ ]  next, prev and replay work
> * [ ]  `-c` history and continue work
> * [ ]  `-d` downloads work
> * [ ]  `-s` syncplay works
> * [ ]  `-q` quality works
> * [ ]  `-v` vlc works
> * [ ]  `-e` select episode works
> * [ ]  `-S` select index works
> * [ ]  `-r` range selection works
> * [ ]  `--skip` ani-skip works
> * [ ]  `--skip-title` ani-skip title argument works
> * [ ]  `--no-detach` no detach works
> * [ ]  `--dub` and regular (sub) mode both work
> * [x]  `-L` language of anime title (jp, jl)
> * [ ]  all providers return links (not necessarily on a single anime, use debug mode to confirm)
> 
> * [x]  `-h` help info is up to date
> * [x]  Readme is up to date
> * [x]  Man page is up to date
> 
> ## Additional Testcases
> * The safe bet: One Piece
> * Episode 0: Saenai Heroine no Sodatekata ♭
> * Unicode: Saenai Heroine no Sodatekata ♭
> * Non-whole episodes: Tensei shitara slime datta ken (ep. 24.5, ep. 24.9)

**Comment by Kamanati** [comment]
.

**Comment by justchokingaround** [comment]
???

**Comment by Kamanati** [comment]
What do you mean by ???

**Comment by justchokingaround** [comment]
why was the 2ani-cli.sh file added and why was the manpage (ani-cli.1) removed???

**Comment by ykhan21** [comment]
Put your printf refactor in another PR.
If you put only the title language change in this PR, it has a higher chance of being accepted.

**Comment by Kamanati** [comment]
Yes, I just removed that

File: ani-cli [hunk 1]
```diff
@@ -27,10 +27,18 @@ nth() {
 }
 
 die() {
-    printf "\33[2K\r\033[1;31m%s\033[0m\n" "$*" >&2
+    printf "\33[2K\r\033[1;31m[error]\033[0m \033[1;37m%s\033[0m\n" "$*" >&2
```
Comment (ykhan21) [inline ani-cli:30]: Remove this from this PR.

File: ani-cli [hunk 2]
```diff
@@ -27,10 +27,18 @@ nth() {
...
 }
 
+info() {
+    printf "\33[2K\r\033[1;32m[Info]\033[0m \033[1;37m%s\033[0m\n" "$*" >&1
+}
+
+notice() {
+    printf "\33[2K\r\033[1;33m[Notice]\033[0m \033[1;37m%s\033[0m\n" "$*" >&1
+}
+
```
Comment (ykhan21) [inline ani-cli:41]: Remove this as well.

File: ani-cli.1 [hunk 3]
```diff
@@ -1,138 +1,558 @@
-.TH "ANI-CLI" "1" "January 2023" "ani-cli" "User Commands"
```
Comment (ykhan21) [inline ani-cli.1:1]: You replaced the manpage with the script.

File: ani-cli (hunk 2 above)
Comment (Kamanati) [reply ani-cli]: Removed

File: ani-cli (hunk 1 above)
Comment (Kamanati) [reply ani-cli]: Removed

File: ani-cli.1 (hunk 3 above)
Comment (Kamanati) [reply ani-cli.1]: Yes I replaced

**Comment by port19x** [comment]
what does this even do?

**Comment by port19x** [comment]
?? I'm closing this

**Comment by Kamanati** [comment]
Why, isn't this a good, bcz the titles are in janpanese in English, you can make it in English right it's use full right

**Pull request closed without merging**